modules/context/api.go

```diff
@@ -351,7 +356,7 @@ func ReferencesGitRepo(allowEmpty bool) func(http.Handler) http.Handler {
 // NotFound handles 404s for APIContext
 // String will replace message, errors will be added to a slice
 func (ctx *APIContext) NotFound(objs ...interface{}) {
```

Comment:
Would be nice to pass here i18n error that issue/PR was not found

Reply:
The reason that I didn't introduce a new i18n translation is: I think the default "not_found" message is clear enough to most users, and we can keep our system small and simple without introducing new texts (and it reduces maintenance work in future).

I am fine with using either a specialized i18n text (new) or a general i18n text (as now). 

Not sure how you think about this: either is fine? or should we introduce specialized i18n texts?